dist/snapcraft.yaml

```diff
@@ -48,10 +46,13 @@ icon: snap/local/freeciv21-client.png
 
 parts:
   freeciv21:
+    override-pull: |
+      snapcraftctl pull
+      snapcraftctl set-version `sed -n '2p' snap/local/AutoRevision.txt`
+      snapcraftctl set-grade "devel"
```

Comment:
Shouldn't this also be computed dynamically?

Reply:
Suggestions on how to do that?

Reply:
The three possible values are `devel`, `candidate`, and `stable`. What about setting `candidate` for rc releases and `stable` for stable releases?

Reply:
I guess I could write an `if` statement that uses `awk` or something to tease out part of the version. `alpah` then gives `devel`, `rc` gives `candidate` and finally the absence of those gives `stable`.